python/rmm/_lib/device_buffer.pyx

```diff
@@ -290,6 +290,13 @@ cdef class DeviceBuffer:
     cdef void* c_data(self):
         return self.c_obj.get()[0].data()
 
+    cdef unique_ptr[device_buffer] release(self):
+        """
+        Releases ownership the data held by this DeviceBuffer.
+        """
+        self.ptr = self.size = self.stream = 0
+        return move(self.c_obj)
```

Comment:
This probably breaks other methods here. Thoughts on putting a trivial `device_buffer` in its place?

Reply:
Can do!